crates/frontend/src/pages/snapshotconfig.rs

```diff
@@ -0,0 +1,57 @@
+use leptos::*;
+use leptos_router::use_params_map;
+use crate::api::get_config_by_version;
+
+#[component]
+pub fn SnapshotConfig() -> impl IntoView {
+    let params = use_params_map();
+    let tenant = params.with(|p| p.get("tenant").cloned().unwrap_or_default());
+    let version = params.with(|p| p.get("version").cloned().unwrap_or_default());
+
+    view! {
+        <div class="p-8">
+            {
+                        let config_resource = create_blocking_resource(
+                            move || (tenant.clone(), version.clone()),
+                            |(tenant, version)| async move {
+                                get_config_by_version(tenant, version).await
+                            },
+                        );
+
+                        view! {
+                            <Suspense fallback=move || view! { <p>"Loading..."</p> }.into_view()>
+                                {move || {
```

Comment:
Can we have some kind of heading here, taking inspiration from the experiment page maybe?

Reply:
Yeah , I will add it.